netkan/netkan/cli.py

```diff
@@ -248,7 +249,8 @@ def download_counter(netkan_remote, ckan_meta, token, key, debug):
         format='[%(asctime)s] [%(levelname)-8s] %(message)s', level=level
     )
     logging.info('Download Counter started at log level %s', level)
-    init_ssh(key, '/home/netkan/.ssh')
+    if netkan_remote.startswith('git@github.com') or ckan_meta.startswith('git@github.com'):
+        init_ssh(key, '/home/netkan/.ssh')
     init_repo(netkan_remote, '/tmp/NetKAN')
```

Comment:
I'm not sure adding a check here is great, especially without a comment as to why. I think populating a dummy string in the `docker-compose.yml` is a safer alternative. If using HTTP(s), it'll never try the key.

Reply:
What if I just remove the `sys.exit(1)` in `init_ssh()`?
A missing ssh key is not a fatal error, especially if a HTTPS URL is used.